### PR TITLE
replace help icon with info icon where appropriate

### PR DESCRIPTION
**Pull request by MichelML**
the info icon is not looking very good on the labeled values right now, i'll see what i can do in vapor for this

### CODE REVIEW

File: src/components/collapsibleContainer/CollapsibleContainer.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export class CollapsibleContainer extends React.Component<ICollapsibleContainerP
     }
 
     const tooltipProps = { placement: 'right', ...this.props.informationTooltip };
-    const svgProps = { svgName: 'help', svgClass: this.getSvgClass() };
+    const svgProps = { svgName: this.props.informationUrl ? 'help' : 'info', svgClass: this.getSvgClass() };
```
Comment (sroy3) [inline src/components/collapsibleContainer/CollapsibleContainer.tsx]: It was `help-no-fill-18` before, will this change anything?

Comment (MichelML) [reply src/components/collapsibleContainer/CollapsibleContainer.tsx]: you can have a look by searching collapsiblecontainer here https://coveo.github.io/react-vapor/info-instead-of-help/ , let me know what you think  

looks fine i wonder if we should add something to white-fill the inside of the icon

Comment (gdostie) [reply src/components/collapsibleContainer/CollapsibleContainer.tsx]: Personally, I prefer when its not white filled. IMHO transparency looks better

Comment (sroy3) [reply src/components/collapsibleContainer/CollapsibleContainer.tsx]: I understand the difference now. The background of the icon is wanted. That was a request from the UX team, so we shouldn't even use `help-no-fill`. The background color is also the right one.

Comment (MichelML) [reply src/components/collapsibleContainer/CollapsibleContainer.tsx]: cool thanks guys

**Pull request merged**